content/03.categorize.md

```diff
@@ -106,18 +106,17 @@ They trained a deep neural network on approximately 100,000 images extracted fro
 The authors used their test set to evaluate when to stop training.
 In other domains, this has resulted in a minimal change in the estimated accuracy [@tag:Krizhevsky2013_nips_cnn], but we recommend the use of an independent test set whenever feasible.
 
-Chest X-rays are a common radiological examination for screening and diagnosis of lung diseases.
-Although hospitals have accumulated a large number of raw radiology images and reports in Picture Archiving and Communication Systems and their related reports in Radiology Information Systems, it is not yet known how to effectively use them to learn the correlation between pathology categories and X-rays.
-In the last few years, deep learning methods showed remarkable results in chest X-ray image analysis [@arxiv:1701.06599; @arxiv:1705.02315].
-However, it is both costly and time-consuming to annotate a large-scale fully-labeled corpus to facilitate data-intensive deep learning models.
-As an alternative, Wang et al. [@arxiv:1705.02315] proposed to use weakly labeled images.
-To generate weak labels for X-ray images, they applied a series of natural language processing (NLP) techniques to the associated chest X-ray radiological reports.
-Specifically, they first extracted all diseases mentioned in the reports using a state-of-the-art NLP tool, then applied a newly-developed negation and uncertainty detection tool (NegBio) to filter negative and equivocal findings in the reports.
-Evaluation on three independent datasets demonstrated that NegBio is highly accurate for detecting negative and equivocal findings (~90% in F-measure, which balances precision and recall [@doi:10.1038/nmeth.3945]).
-These highly-accurate results meet the need to generate a corpus with weak labels, which serves as a solid foundation for the later process of image classification.
-The resulting dataset [@url:https://nihcc.app.box.com/v/ChestXray-NIHCC] consists of 112,120 frontal-view chest X-ray images from 30,805 patients, and each image is associated with one or more weakly-labeled pathology category (e.g. pneumonia and cardiomegaly) or "normal" otherwise.
-Further, Wang et al.
-[@arxiv:1705.02315] used this dataset with a unified weakly-supervised multi-label image classification framework, to detect common thoracic diseases.
+Although rich clinical information is stored in EHRs, manually annotating them is not only costly and time consuming but also requires domain knowledge.
```

Comment:
Suggested:
```
Rich clinical information is stored in EHRs.
However, manually annotating a large set requires experts and is time consuming.
For chest X-ray studies, a radiologist usually spends a few minutes per example. 
Generating the number of examples needed for deep learning is infeasibly expensive.
Instead, researchers may benefit from using text mining to generate annotations [@doi:10.3115/1572392.1572411], even if those annotations are of modest accuracy.
Wang et al. [@arxiv:1705.02315] proposed to build predictive DL models through the use of images with *weak labels*.
Such labels are automatically generated and not verified by humans, so they may be noisy or incomplete.
... (continue as written) ...


```

I think this is a bit more compact and leads the reader through a bit more.